src/definitions/device_key/mod.rs

```diff
@@ -7,27 +50,37 @@ pub mod cose_key;
 pub use cose_key::CoseKey;
 pub use cose_key::EC2Curve;
 
+/// Represents information about a device key.
 #[derive(Clone, Debug, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct DeviceKeyInfo {
+    /// The device key.
     pub device_key: CoseKey,
+
+    /// Optional key authorizations.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub key_authorizations: Option<KeyAuthorizations>,
+
+    /// Optional key information.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub key_info: Option<BTreeMap<i128, CborValue>>,
 }
 
 #[derive(Clone, Serialize, Deserialize, Debug, Default)]
 #[serde(rename_all = "camelCase")]
 pub struct KeyAuthorizations {
+    /// The namespaces associated with the key. This field is optional and will
+    /// be skipped during serialization if it is [None].
     #[serde(skip_serializing_if = "Option::is_none", rename = "nameSpaces")]
     pub namespaces: Option<NonEmptyVec<String>>,
-    #[serde(skip_serializing_if = "Option::is_none")]
+
+    /// The data elements associated with the key. This field is optional and will
+    /// be skipped during serialization if it is [None].
     pub data_elements: Option<NonEmptyMap<String, NonEmptyVec<String>>>,
```

Comment:
Similar question as above. Need to update the comments if we intend to skip serialization for `Option::None` values.